tests/driver_ws281x/Makefile

```diff
@@ -0,0 +1,21 @@
+BOARD ?= atmega328p
+include ../Makefile.tests_common
+
+ifeq (atmega328p,$(BOARD))
+  PIN ?= GPIO_PIN(PORT_B, 0)
+  N ?= 10
+else
+  ifneq (,$(filter arduino-nano arduino-uno,$(BOARD)))
+    PIN ?= GPIO_PIN(PORT_D, 2) # Digital pin 2 on Arduino Uno / Nano
+  else
```

Comment:
What's the significance of those pins?

Reply:
It made my life with testing easier ;-). I dropped it now.